.env.example

```
@@ -0,0 +1,2 @@
+VITE_API_URL=https://api.stg.idp.gistory.me # https://api.idp.gistory.me when production
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)